automation/label.yml

```diff
@@ -9,4 +11,4 @@ jobs:
     steps:
     - uses: actions/labeler@v2
```

Comment:
Labeler V2, capital as described the Person, subject under person small or (), eTc. As per agreement Person state must be CaPiTaL. (CPTL) in a Manner oF senTeNcE grammar and codes used sucHLikE gitHub codes. SamE aS I wriTE, before I looked untO diveRsion thAts how I typE, nOW small cap follOwEd bY big caps unto MucH cLEareR DirecTivEs.